src/js/editor/edit-state.js

```diff
@@ -115,6 +130,21 @@ class EditState {
     return post.markupsInRange(range);
   }
 
+  _readSectionAttributes(sections) {
+    return sections.reduce((sectionAttributes, s) => {
+      let attributes = s.isNested ? s.parent.attributes : s.attributes;
+      Object.keys(attributes || {}).forEach(attrName => {
+        let camelizedAttrName = attrName.replace(/^data-md-/, '');
+        let attrValue = attributes[attrName];
+        sectionAttributes[camelizedAttrName] = sectionAttributes[camelizedAttrName] || [];
+        if (!sectionAttributes[camelizedAttrName].includes(attrValue)) {
```

Comment:
There's a function `contains` in [array-utils.js](https://github.com/bustle/mobiledoc-kit/blob/master/src/js/utils/array-utils.js#L151) that should be used instead of `Array.includes` as the latter is not supported for IE11. The tests on Sauce Labs should catch this (when they are working again) but just a heads up.

Reply:
Thanks @YoranBrondsema! Fixed.